modules/tinymce/src/core/test/ts/browser/EditorRemoveTest.ts

```diff
@@ -38,7 +38,7 @@ describe('browser.tinymce.core.EditorRemoveTest', () => {
     await McEditor.pFromHtml('<textarea></textarea>', {
       ...settings,
       setup: (editor: Editor) => {
-        editor.on('LoadContent', () => {
+        editor.on('PreInit', () => {
```

Comment:
Why was this changed?

Reply:
Since it patches over the style sheet loader it needs to be done before the actual css starts loading.